src/main/java/org/frc5687/rapidreact/OI.java

```diff
@@ -24,10 +32,11 @@ public OI() {
 
         _translation = new Joystick(0);
         _rotation = new Joystick(1);
+        resetNavX = new JoystickButton(_translation, 5);
     }
 
     public void initializeButtons(DriveTrain driveTrain) {
-        //There's nothing to init here
+        resetNavX.whenPressed(new ResetNavX(driveTrain));   
```

Comment:
Do we need a command for this? If not, we can change this to `restNavX.whenPressed(driveTrain::resetNavX)` thanks to the help of the reference operator `::`. They are similar to lambda operators `->`.